lib/utils/set-env.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+const path = require('path');
+const fs = require('fs');
+
+module.exports = function setEnv(rootDirectory = process.cwd()) {
+  const NODE_ENV = process.env.NODE_ENV;
+  const dontenvPath = path.join(rootDirectory, '.env');
+
+  // https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use
+  const dotenvFiles = [
+    NODE_ENV && `${dontenvPath}.${NODE_ENV}.local`,
+    NODE_ENV && `${dontenvPath}.${NODE_ENV}`,
+    // Don't include `.env.local` for `test` environment
+    // since normally you expect tests to produce the same
+    // results for everyone
+    NODE_ENV !== 'test' && `${dontenvPath}.local`,
```

Comment:
Do we have any use cases for a `.local` file? If not, I'd be in favor of removing the code that makes it work.